network/msgCompressor_test.go

```diff
@@ -52,30 +52,15 @@ func TestCheckCanCompress(t *testing.T) {
 	partitiontest.PartitionTest(t)
```

Comment:
Ideally match the test name to the new function name

Reply:
done